### PR TITLE
dungnd/feature/31131

### PR DESCRIPTION
**Pull request by dungnd193**
Connect BE and build login normal

### CODE REVIEW

File: api-client/axios-client.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const axiosClient = axios.create({
+  baseURL: "http://localhost:3001",
```
Comment (daitd58) [inline api-client/axios-client.ts]: Push this string to env variable: NEXT_PUBLIC_API_URL

File: api-client/axios-client.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const axiosClient = axios.create({
+  baseURL: "https://kryptohub-be.herokuapp.com",
```
Comment (daitd58) [inline api-client/axios-client.ts]: push this link to env file

Comment (dungnd193) [reply api-client/axios-client.ts]: Resolved

File: api-client/axios-client.ts (hunk 1 above)
Comment (dungnd193) [reply api-client/axios-client.ts]: Resolved

**Pull request merged**